js/tsmlfe.js

```diff
@@ -73,3 +77,8 @@ function toggleAdditionalInfoDisplay(x, y) {
 		x.style.display = "none";
 	}
 }
+
+function inactive(x) {
+	x = document.getElementById(x);
+	x.value = + " (Inactive)";
```

Comment:
did you mean to do `+=` here

Reply:
Must have been half asleep when I did that...